flake8_implicit_str_concat.py

```diff
@@ -35,7 +35,8 @@ def _implicit(file_tokens: Iterable[tokenize.TokenInfo]) -> Iterable[_ERROR]:
             *a.end,
             "ISC001 implicitly concatenated string literals on one line"
             if a.end[0] == b.start[0]
-            else "ISC002 implicitly concatenated string literals over continuation line",
+            else "ISC002 implicitly concatenated string literals "
+            "over continuation line",
```

Comment:
what made this change?

Reply:
I made it manually because it was over the Flake8 limit:

```
./flake8_implicit_str_concat.py:38:89: E501 line too long (89 > 88 characters)
```

Would it be better to break it at another point? Or add `#noqa: E501`?

Reply:
The split seems fine to me.